ravindu14/SliitSpmProject
# Feature/task id #1156 form validation

how to test
1. run "nom install"
2. start the application by "ng serve" command
3. navigate to "http://localhost/4200/tables" url
4. click on save button without filling name,nic and upload form field (an error message should be displayed asking to fill all the fields)
